submissions/smillims/memory-pair-game/js/script.js

```diff
@@ -1,89 +1,110 @@
 'use strict';
 
-const allCards = document.querySelectorAll('.flipper');
+const cards = document.querySelector('.cards');
 
-const cardPosition = 12;
-const allPairCards = 6;
-let currentPair = 0;
-let hasFlippedCard = false;
 let lockCard = false;
 let firstCard = '';
 let secondCard = '';
 
-function flipCards() {
-	if(lockCard){
-		return;
-	};
-	if (this === firstCard){
+const images = [
+	'bentley',
+	'amazon',
+	'google',
+	'intel',
+	'nike',
+	'telegram'
+];
+
+const allImg = [...images, ...images];
```

Comment:
why did you use such an abbreviation?
img - one image. But here a lot of images.
Avoid using abbrevations  it will prevent disinformation